src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -136,12 +152,37 @@ class AbstractIntegrationDetailedView<
     throw new Error('Not implemented');
   }
 
+  renderButton(
+    _disabledFromFeatures: boolean, //from the feature gate
```

Comment:
@NisanthanNanthakumar why are the inputs to this function prefixed with an underscore? That usually suggests the variable is unused but that doesn't seem to be the case.